docs/user/technical-reference/03-10-buildless-serverless.md

```diff
@@ -0,0 +1,48 @@
+# Build-less mode of Serverless
+
+## Overview: Moving Towards Buildless Mode
+
+From the beginning, Kyma Serverless has aimed to accelerate the development of fast prototypes by allowing users to focus on business logic rather than containerization and Kubernetes deployment. Our goal is to remove operational barriers so developers can iterate quickly and efficiently.
+
+With the introduction of Buildless mode, we are taking this vision further. By eliminating the image build step in the Kyma runtime, we significantly shorten the feedback loop during prototype development. In buildless mode, instead of building and pushing custom function images into in-cluster registry, your code and dependencies are simply mounted into Kyma-provided runtime images. This approach positions Kyma Serverless as more efficient development tool, enabling even faster iteration. Additionally, it eliminates the architectural complexities and limitations of deploying serverless Functions on Kubernetes.
```

Comment:
```suggestion
With the introduction of buildless mode, we are taking this vision further. We significantly shorten the feedback loop during prototype development by eliminating the image build step in Kyma runtime. In buildless mode, instead of building and pushing custom function images into the in-cluster registry, your code and dependencies are mounted into Kyma-provided runtime images. This approach positions Kyma Serverless as a more efficient development tool, enabling even faster iteration. Additionally, it eliminates the architectural complexities and limitations of deploying Serverless Functions on Kubernetes.
```